gitpractice/javascript/task8.js

```diff
@@ -0,0 +1,10 @@
+/*http://www.codewars.com/kata/word-count*/
+function countWords(str) {
+    var string = str;
```

Comment:
You can directly use str, no need to assign it to other variable and use it

Reply:
yes i used it directly in updated tasks